apps/connect/package.copy.json

```diff
@@ -3,7 +3,7 @@
   "productName": "Powerhouse-Connect",
   "version": "4.1.0-dev.100",
   "description": "Powerhouse Connect",
-  "main": "./dist/index.html",
+  "main": "dist/index.html",
```

Comment:
I've never really understood what the deal is with the paths here. what is the difference with vs without the ./ ? Not a problem, just curious